localization/english/wc_uther_events_l_english.yml

```diff
@@ -3,4 +3,28 @@
  uther_events.001.d:0 "By the light!\n\nCrisis has befallen Lordaeron. Prince Arthas has returned from Northrend, consumed by madness. He has slain his own father and now seeks to bring destruction to Lordaeron and beyond.\n\nHow could this happen? I was his mentor, it was my duty to instill in him the virtues of the Silver Hand. Even when he crossed the line and purged the innocent at Stratholme, he still sought to protect the kingdom. All rulers must make difficult decisions to protect their people. But now, I must make my own.\n\nMy fellow paladins have gathered, and now I am to lead them in destroying the undead Scourge. But with Arthas at it's helm, that will mean killing him..."
  uther_events.001.a:0 "No, I cannot. I shall discard my hammer, forever."
  uther_events.001.b:0 "Damn it, I will put you down, Arthas."
- uther_events.001.c:0 "Lordaeron must be protected at any cost."
\ No newline at end of file
+ uther_events.001.c:0 "Lordaeron must be protected at any cost."
+ uther_tooltip:0 "You will try and convince Arthas otherwise, or abandon the investigation."
```

Comment:
```suggestion
 uther_tooltip:0 "You will try and convince [GetGlobalVariable('arthas_character').Char.GetFirstName] otherwise, or abandon the investigation."
```